site/specs/numbers.json

```diff
@@ -5677,47 +5756,45 @@
             "content": {
               "application/xml": {
                 "examples": {
                   "example": {
-                    "description": "example",
-                    "value": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n<ImportTnOrder>\n <OrderCreateDate>2018-01-09T02:58:04.615Z</OrderCreateDate>\n <AccountId>9900012</AccountId>\n <CreatedByUser>sjm</CreatedByUser>\n <OrderId>bf1305b8-8998-1111-2222-51ba3ce52d4e</OrderId>\n <LastModifiedDate>2018-01-09T02:58:05.298Z</LastModifiedDate>\n <SiteId>65487</SiteId>\n <SipPeerId>885544</SipPeerId>\n <Subscriber>\n <Name>First</Name>\n <ServiceAddress>\n <HouseNumber>11235</HouseNumber>\n <StreetName>Back</StreetName>\n <City>Denver</City>\n <StateCode>CO</StateCode>\n <Zip>27541</Zip>\n <County>Canyon</County>\n </ServiceAddress>\n </Subscriber>\n <LoaAuthorizingPerson>The Authguy</LoaAuthorizingPerson>\n <LoaType>CARRIER</LoaType>\n <TelephoneNumbers>\n <TelephoneNumber>2106078250</TelephoneNumber>\n <TelephoneNumber>2109678273</TelephoneNumber>\n <TelephoneNumber>2109678331</TelephoneNumber>\n <TelephoneNumber>2109678337</TelephoneNumber>\n <TelephoneNumber>2266401468</TelephoneNumber>\n </TelephoneNumbers>\n <ProcessingStatus>PARTIAL</ProcessingStatus>\n <Errors>\n <Error>\n <Code>7518</Code>\n <Description>Telephone Number Not Active.</Description>\n <TelephoneNumbers>\n <TelephoneNumber>2262665583</TelephoneNumber>\n </TelephoneNumbers>\n </Error>\n </Errors>\n </ImportTnOrder>\n"
+                    "value": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n<ImportTnOrder>\n\t <OrderCreateDate>2018-01-09T02:58:04.615Z</OrderCreateDate>\n\t <AccountId>9900012</AccountId>\n\t <CreatedByUser>sjm</CreatedByUser>\n\t <OrderId>bf1305b8-8998-1111-2222-51ba3ce52d4e</OrderId>\n\t <LastModifiedDate>2018-01-09T02:58:05.298Z</LastModifiedDate>\n\t <SiteId>65487</SiteId>\n\t <SipPeerId>885544</SipPeerId>\n\t <Subscriber>\n\t\t <Name>First</Name>\n\t\t <ServiceAddress>\n\t\t\t <HouseNumber>11235</HouseNumber>\n\t\t\t <StreetName>Back</StreetName>\n\t\t\t <City>Denver</City>\n\t\t\t <StateCode>CO</StateCode>\n\t\t\t <Zip>27541</Zip>\n\t\t\t <County>Canyon</County>\n\t\t </ServiceAddress>\n\t </Subscriber>\n\t <LoaAuthorizingPerson>The Authguy</LoaAuthorizingPerson>\n\t <LoaType>CARRIER</LoaType>\n\t <TelephoneNumbers>\n\t\t <TelephoneNumber>2106078250</TelephoneNumber>\n\t\t <TelephoneNumber>2109678273</TelephoneNumber>\n\t\t <TelephoneNumber>2109678331</TelephoneNumber>\n\t\t <TelephoneNumber>2109678337</TelephoneNumber>\n\t\t <TelephoneNumber>2266401468</TelephoneNumber>\n\t </TelephoneNumbers>\n\t <ProcessingStatus>PARTIAL</ProcessingStatus>\n\t <Errors>\n\t\t <Error>\n\t\t\t <Code>7518</Code>\n\t\t\t <Description>Telephone Number Not Active.</Description>\n\t\t\t <TelephoneNumbers>\n\t\t\t\t <TelephoneNumber>2262665583</TelephoneNumber>\n\t\t\t </TelephoneNumbers>\n\t\t </Error>\n\t </Errors>\n </ImportTnOrder>\n"
```

Comment:
do we want to use tabs or spaces here?

Reply:
fixed

Reply:
That was a genuine question, I wasn't sure whether the docsite wanted tabs or spaces, or does it support both?

Reply:
It does seem like in many of the XML examples in here we're mixing tabs and spaces.  This one has a tab and then 1 space before the next XML tag, below you added a tab before 4 spaces.  It would make sense to use one or the other for clarity.  It may not matter to the user, depending on how the docsite generator treats spaces.